adapter/internal/discovery/xds/server.go

```diff
@@ -228,6 +228,7 @@ func UpdateAPI(apiContent config.APIContent) {
 
 	if apiContent.APIType == mgw.HTTP {
 		mgwSwagger = operator.GetMgwSwagger(apiContent.APIDefinition)
+		mgwSwagger.SetID(apiContent.UUID)
```

Comment:
Set API UUID for mgwSwagger.id from here.
For WS APIs, API UUID is mgwSwagger.id inside the function operator.GetMgwSwaggerWebSocket
Setting the same for HTTP APIs.